_data/projects/cherrybomb.yml

```diff
@@ -0,0 +1,10 @@
+name: Cherrybomb
+desc: Stop half-done API specifications - Cherrybomb is a CLI tool that helps you avoid undefined user behaviour by validating your API specifications.
+site: https://github.com/blst-security/cherrybomb
+tags:
+  - API-security
```

Comment:
@yigblst We need to fix these errors, please change names accordingly.
  - Tag 'API-security' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -
  - Tag 'Rust' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -
  - Tag 'CLI' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -